ArduPlane/GCS_Mavlink.cpp

```diff
@@ -1455,7 +1455,8 @@ void GCS_MAVLINK_Plane::handleMessage(mavlink_message_t* msg)
         case MAV_CMD_COMPONENT_ARM_DISARM:
             if (is_equal(packet.param1,1.0f)) {
                 // run pre_arm_checks and arm_checks and display failures
-                if (plane.arm_motors(AP_Arming::MAVLINK)) {
+                const bool do_arming_checks = !is_equal(packet.param2,2989.0f);
```

Comment:
Ideally as an ardupilot.xml enum. Failing that a #define in GCS.h. But the proper place is a enum with exactly one value :)